examples/http-disable-chunking/http_disable_chunking.bal

```diff
@@ -4,49 +4,46 @@ import ballerina/log;
 //The HTTP client's chunking behaviour can be configured as auto, always, or never.
 //In this example, it is set to as never, which means that chunking never happens irrespective of how it is specified
 //in the request. When chunking is set to auto, chunking is done as specified in the request.
-endpoint http:Client clientEndpoint {
-    url: "http://localhost:9090",
+http:ClientEndpointConfig clientEndpointConfig = {
     chunking: http:CHUNKING_NEVER
 };
 
-service<http:Service> chunkingSample bind { port: 9092 } {
+service chunkingSample on new http:Listener(9092) {
 
     @http:ResourceConfig {
         path: "/"
     }
     //Parameters include a reference to the caller endpoint and an object with the request data.
-    invokeEndpoint(endpoint caller, http:Request req) {
+    resource function invokeEndpoint(http:Caller caller, http:Request req) {
+        http:Client clientEndpoint = new("http://localhost:9090", config = clientEndpointConfig);
         //Create a new outbound request and set the payload.
         http:Request newReq = new;
         newReq.setPayload({ "name": "Ballerina" });
         var result = clientEndpoint->post("/echo/", newReq);
-        match result {
-            http:Response clientResponse => {
-                //send the response back to the caller.
-                var result = caller->respond(clientResponse);
-                if (result is error) {
-                   log:printError("Error sending response", err = result);
-                }
+        if (result is http:Response) {
+            //send the response back to the caller.
+            var resp = caller->respond(result);
```

Comment:
shall we change the variable name as this is not a response? result would be suitable